File: Nelly Ndinda.cpp
#include <iostream>
#include <string>
using namespace std;

class roommate {
public: 
    string name;
    int age;
    string hobby;
    string gender;
    string skincolor;
    string job;
};
int main() {
    roommate roommate1;
    
    roommate1.name = "Cynthia Mueni";
    roommate1.age = 26;
    roommate1.hobby = "cooking,horse riding, dancing";
    roommate1.gender = "Female";
    roommate1.skincolor = "lightskin";
    cout<< "Name: " <<roommate1.name << endl;
    cout << "Age: " <<roommate1.age << endl;
    cout << "Hobby: " << roommate1.hobby << endl;
    cout << "gender:" << roommate1.gender << endl;
    cout << "skincolor:" << roommate1.skincolor << endl;     
    cout << endl;
     
     return 0;
};

File: Nelly Ndinda .py

class Roommate:
    def __init__(self, name, age, hobby, gender, skincolour, job):
        self.name =name
        self.age = age
        self.hobby = hobby
        self.gender = gender
        self.skincolour = skincolour
        self.job= job
    
    def display_info(self):
        print("\n--- Roommate Information ---")
        print(f"Name: {self.name}")
        print(f"age: {self.age}")
        print(f"hobby: {self.hobby}")
        print(f"gender: {self.gender}")
        print(f"skincolour: {self.skincolour}")
        print(f"job: {self.job}")
Roommate1 = Roommate("Cynthia Mueni", 26, "horse riding", "female","light skin" ,"supervisor at jaza supermarket")
Roommate1.display_info()